beets/autotag/hooks.py

```diff
@@ -157,9 +158,10 @@ def __init__(self, title, track_id, artist=None, artist_id=None,
                  medium_total=None, artist_sort=None, disctitle=None,
                  artist_credit=None, data_source=None, data_url=None,
                  media=None, lyricist=None, composer=None, composer_sort=None,
-                 arranger=None, track_alt=None):
+                 arranger=None, track_alt=None, work_id=None):
```

Comment:
I would put the `work_id=None` into the same place it's put in all the other contexts (ie., after `track_id`). Other places *should* be using `artist=…, …` to specify the artist etc. specifically. If they don't and thus break by this, we've found places that need fixing.